opm/autodiff/BlackoilPropsAd.cpp

```diff
@@ -935,5 +935,14 @@ namespace Opm
         OPM_THROW(std::logic_error, "BlackoilPropsAd class does not support this functionality.");
     }
 
```

Comment:
The doc for this function (and all the others, since it is the same) is not correct, the descriptions of `saturation` and `pc` do not make sense (cell indices?).